pages/glossary/hovered.md

```diff
@@ -0,0 +1,6 @@
+--
+title: Hovered
+key: hovered
+--
+
+An element is hovered (by a pointing device) when that element matches the [`:hover` pseudo-class](https://drafts.csswg.org/selectors-4/#hover-pseudo).
```

Comment:
Why is this definition different from the focus one? Why not say the same about focus, that an element is focused when it matches the `:focus` pseudo-class?